README.rst

```diff
@@ -0,0 +1,144 @@
+AlloyDB for PostgreSQL for LangChain
```

Comment:
Can we match the exact format in https://github.com/googleapis/langchain-google-cloud-sql-pg-python/blob/main/README.rst?plain=1

Reply:
matched